changes/issue-7864-host-orbit-info

```diff
@@ -0,0 +1,3 @@
+- Added ingestion of host orbit version from `orbit_info` osquery extension table.
```

Comment:
@lucasmrod @roperzh the orbit_info table is not going away, correct? We are just not using it like before.

Reply:
Correct. It's still "served" by Orbit, it's just that Fleet stopped using it (for the token ingestion).

Reply:
that is correct, it's there and it even has more info! but we're not using it to ingest `host_device_auth`, and thus not sending it as a distributed query anymore